callstats-java-sdk/src/test/java/io/callstats/sdk/CallStatsTest.java

```diff
@@ -22,7 +22,7 @@
 	CallStatsInitListener listener;
 	
 	/** The endpoint info. */
-	EndpointInfo endpointInfo;
+	ServerInfo serverInfo;
 	
 	/** The app id. */
 	int appId = 497346896;
```

Comment:
Remove 'appId' and 'appSecret' values and try to read it from either an environment variables or a config files which does not reside in the repo.